api/src/services/summary-service.test.ts

```diff
@@ -490,7 +490,7 @@ describe('SummaryService', () => {
         ]);
 
       const getRules = sinon.stub(service, 'getValidationRules').resolves('');
-      const validate = sinon.stub(service, 'validateXLSX').resolves({});
+      const validate = sinon.stub(service, 'validateXLSX_summary').resolves({});
```

Comment:
revert ... make a ticket for techdebt to address combining .. maybe along with status ticket